core/util.go

```diff
@@ -364,11 +364,13 @@ func GetBuildHost() (retID string) {
 	return
 }
 
-// UniqueNames returns the set of all unique names in the input.
+// UniqueNames returns the set of all unique names in the input after. The
+// returned names will all be lowercased and will have been checked against each
+// other in their lowercase form..
 func UniqueNames(names []string) (unique []string) {
```

Comment:
Maybe it's because I'm pre-coffee, but I had to look through `certificate-authority.go` a couple times to catch that UniqueNames is responsible for lowercasing. Maybe we should rename the method?